lib/everypolitician/popolo/collection.rb

```diff
@@ -6,8 +6,21 @@ class Collection
       attr_reader :documents
       attr_reader :popolo
 
+      # set the class that represents individual items in the
+      # collection
+      def Collection.entity_class(entity)
+        @entity_class = entity
+      end
+
+      # we need to do this slightly awful looking thing becuase
+      # we want a class instance variable and not an instance
+      # variable
+      def entity_class
+        self.class.instance_variable_get :@entity_class
```

Comment:
There might be a less clunky looking way to do this but this is the best my ruby could manage.